mne/viz/tests/test_raw.py

```diff
@@ -106,9 +106,10 @@ def test_plot_raw_filtered():
     assert_raises(ValueError, raw.plot, lowpass=1, highpass=1)
     assert_raises(ValueError, raw.plot, lowpass=1, filtorder=0)
     assert_raises(ValueError, raw.plot, clipping='foo')
-    raw.plot(lowpass=1, clipping='transparent')
-    raw.plot(highpass=1, clipping='clamp')
-    raw.plot(highpass=1, lowpass=2)
+    with warnings.catch_warnings(record=True):  # bad proj
+        raw.plot(lowpass=1, clipping='transparent')
+        raw.plot(highpass=1, clipping='clamp')
+        raw.plot(highpass=1, lowpass=2)
```

Comment:
I think i'd rather hack the _get_raw function to fake proper projections. eg. recompute them with an SVD or normalize them to 1. This applies to other tests.